tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "outDir": "dist",
     "lib": ["ES6"],
     "baseUrl": ".",
+    "incremental": true,
```

Comment:
What does this do?

Reply:
Enable incremental compilation by reading/writing information from prior compilations to a file on disk.